src/axolotl/train.py

```diff
@@ -115,6 +115,11 @@ def terminate_handler(_, __, model):
     badge_markdown = """[<img src="https://raw.githubusercontent.com/OpenAccess-AI-Collective/axolotl/main/image/axolotl-badge-web.png" alt="Built with Axolotl" width="200" height="32"/>](https://github.com/OpenAccess-AI-Collective/axolotl)"""
     transformers.modelcard.AUTOGENERATED_TRAINER_COMMENT += f"\n{badge_markdown}"
 
+    if getattr(cfg, "axolotl_config_path"):
```

Comment:
should `axolotl_config_path` be explicit? or should we be setting it from the input path for the train call?

Reply:
The reason for this line of code is actually because the config isn't passed in the same way in e2e unit tests which is not how users use axolotl I think.  

I'm not sure I see a strong reason to change it yet?

Reply:
oh, nvm, I see now, this is just reusing the existing var we setup for wandb. 👍